src/Components/BookDetail/Reviews/Reviews.jsx

```diff
@@ -0,0 +1,28 @@
+import BookReview from "./BookReview/BookReview"
+function Reviews({reviews}) {
```

Comment:
maybe add a space here as well for consistency

Reply:
Done, whitespace added.